modules/UI/videolayout/SmallVideo.js

```diff
@@ -548,43 +523,19 @@ SmallVideo.prototype.showRaisedHandIndicator = function (show) {
         return;
     }
 
-    var indicatorSpan = this.getIndicatorSpan({
-        id: 'raisehandindicator',
+    let indicatorSpanId = "raisehandindicator";
```

Comment:
This method and the dominant speaker method are the same. The only differences are the parameters of UIUtil.getVideoThumbnailIndicatorSpan. Maybe it will be good to combine them? WDYT?

NOTE: I know that this is old code but I've just noticed it. We can do that with another PR as well.